build.gradle

```diff
@@ -1,21 +1,53 @@
 plugins {
-    id 'java'
-    id 'org.springframework.boot' version '2.2.7.RELEASE'  apply false
+    id "org.springframework.boot" version "2.4.1" apply false
+    id "io.spring.dependency-management" version "1.0.10.RELEASE" apply false
+    id "org.springdoc.openapi-gradle-plugin" version "1.3.0"
```

Comment:
this plugin should be disabled everywhere except the examples module, as you don't generate anything in the starter module

Reply:
Could you specify the exact ones, please?

- springframework.boot
- dependency-management
- openapi-gradle-plugin

Thanks!